### DISCUSSION TITLE
Unable to access the GitHub page generated using al-folio using eduroam or university Wi-Fi

### DISCUSSION THREAD

**Discussion by Aviemathelec1995** [alshedivat/al-folio#2807] [Q&A]
Hi, 
I have created my webpage https://aviemathelec1995.github.io/ using al-folio. 
I can access this webpage from any Wi-Fi other than my university Wi-Fi or eduroam network.
Whenever I am trying to access the webpage using eduroam or my university Wi-Fi it is showing "This site can’t be reached".
Is there any way it can be resolved through the repository?

Similar issue was also raised here by another user -https://github.com/orgs/community/discussions/133239 .

**Comment by george-gca** [comment]
Duplicate of #2276. Check the discussion there, see if it helps.
